client/state/data-layer/wpcom-http/actions.js

```diff
@@ -46,6 +46,16 @@ export const http = (
 ) => {
 	const version = apiNamespace ? { apiNamespace } : { apiVersion };
 
+	if ( process.env.NODE_ENV !== 'production' ) {
```

Comment:
any reason not to gate this for development?

right now I believe it will appear in testing too

Reply:
no great reason. changed to dev